packages/design-system-tailwind-preset/src/typography.ts

```diff
@@ -5,29 +5,18 @@ export const typography = {
     's-heading-lg': 'var(--typography-s-heading-lg-font-size)',
```

Comment:
In this file we are removing all typography variant classnames with a font-weight suffix. e.g. -regular, -medium, -bold.